### PR TITLE
Add a test status section to display testgrid status.

### PR DESCRIPTION
**Pull request by siyuanfoundation**
Please read https://github.com/etcd-io/etcd/blob/main/CONTRIBUTING.md#contribution-flow.

Related to https://github.com/etcd-io/etcd/issues/13167.

### CODE REVIEW

**Comment by siyuanfoundation** [comment]
Not sure README.md is the right place for this, but it seems to be useful information to display somewhere on the website. @ahrtr @serathius

File: README.md [hunk 1]
```diff
@@ -133,6 +133,19 @@ Now it's time to dig into the full etcd API and other guides.
 [security]: https://etcd.io/docs/latest/op-guide/security
 [tuning]: https://etcd.io/docs/latest/tuning
 
+## Test Status
```
Comment (serathius) [inline README.md:136]: Would it be better to have it in CONTRIBUTING.md? or one of `Documentation/contributor-guide`

Comment (jmhbnz) [reply README.md:136]: My vote would be `CONTRIBUTING.md` as a new heading below the "Create a pull request" header: https://github.com/etcd-io/etcd/blob/main/CONTRIBUTING.md#create-a-pull-request

Comment (ahrtr) [reply README.md:136]: Suggest to add one more doc something like `test-status.md` under `Documentation/contributor-guide`, and then add an item under [Find something to work on](https://github.com/etcd-io/etcd/blob/main/CONTRIBUTING.md#find-something-to-work-on) to point to the doc.

Either way works for me. The goal is to have people periodically check and drive to fix all the flaky test. Leave it to @jmhbnz to coordinate this via the bi-weekly bug triage meeting?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -43,6 +43,34 @@ If any of aforementioned labels don't have unassigned issues, please [contact] o
 [maintainers]: https://github.com/etcd-io/etcd/blob/main/OWNERS
 [priority/important]: https://github.com/search?type=issues&q=org%3Aetcd-io+state%3Aopen++label%3A%22priority%2Fimportant%22
 
+### Flaky Tests
```
Comment (ahrtr) [inline CONTRIBUTING.md:46]: pls add an entry in the table of contents.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -43,6 +43,34 @@ If any of aforementioned labels don't have unassigned issues, please [contact] o
...
+| presubmit e2e-amd64  | [![sig-etcd-presubmits/pull-etcd-e2e-amd64](https://testgrid.k8s.io/q/summary/sig-etcd-presubmits/pull-etcd-e2e-amd64/tests_status?style=svg)](https://testgrid.k8s.io/q/summary/sig-etcd-presubmits/pull-etcd-e2e-amd64)  |
+| presubmit unit-test  | [![sig-etcd-presubmits/pull-etcd-unit-test](https://testgrid.k8s.io/q/summary/sig-etcd-presubmits/pull-etcd-unit-test/tests_status?style=svg)](https://testgrid.k8s.io/q/summary/sig-etcd-presubmits/pull-etcd-unit-test)  |
+| presubmit verify     | [![sig-etcd-presubmits/pull-etcd-verify](https://testgrid.k8s.io/q/summary/sig-etcd-presubmits/pull-etcd-verify/tests_status?style=svg)](https://testgrid.k8s.io/q/summary/sig-etcd-presubmits/pull-etcd-verify)  |
+| postsubmit build     | [![sig-etcd-postsubmits/post-etcd-build](https://testgrid.k8s.io/q/summary/sig-etcd-postsubmits/post-etcd-build/tests_status?style=svg)](https://testgrid.k8s.io/q/summary/sig-etcd-postsubmits/post-etcd-build)  |
+
+If you find any flaky tests on testgrid, please
+
+1. open an issue with the `type/flake` label.
```
Comment (ahrtr) [inline CONTRIBUTING.md:63]: Do we expect each contributor raise issue for the flaky test? Or should we take care of it? @jmhbnz

**Review by jmhbnz — APPROVED** [review]
Overall LGTM - Just a few nitpicks below if you have time.

File: CONTRIBUTING.md (hunk 3 above)
Comment (ahrtr) [reply CONTRIBUTING.md:63]: > We should probably suggest the check for existing issues first though.

Agreed.

File: CONTRIBUTING.md (hunk 2 above)
Comment (siyuanfoundation) [reply CONTRIBUTING.md]: done

**Review by ahrtr — APPROVED** [review]
lgtm

Thanks

**Pull request merged**